metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -257,6 +262,22 @@ final class Doctor(
 
   def allTargets(): List[ScalaTarget] = buildTargets.all.toList
 
+  private def selectedBuildToolMessage(): Option[String] = {
+    tables.buildTool.selectedBuildTool().map { value =>
+      s"Your ${value} build definition has been imported "
+    }
+  }
+
+  private def selectedImportBuildMessage(): Option[String] = {
+    import scala.concurrent.duration._
+    tables.dismissedNotifications.ImportChanges.whenExpires().map { expire =>
```

Comment:
```suggestion
    tables.dismissedNotifications.ImportChanges.whenExpires().map { expiration =>
```
Nit on wording

Reply:
Thanks for testing it and catching this problem!
I will not display this reset link in case `isHttpEnabled=true` as it will be simplest for now.

Maybe in that case add additional line like `To reset those choices execute command: metals.choice-reset` but would users understand that? As probably commands in LSP are mapped differently in different editors.

Reply:
Yea it's either have to be a very vague message about sending the command, or just don't mention it. Then we'll just have to make sure it's documented well in the docs.

Reply:
Or maybe say something more general like: `You can reset your choices with command 'Reset Choice'` and assume users of different editors will be able to map it to their editor command?

Reply:
Ok I am not adding any additional message, we can add it into docs :+1: